html/elements/legend.json

```diff
@@ -24,29 +24,76 @@
               "version_added": "6"
             },
             "opera": {
-              "version_added": true
+              "version_added": "≤12.1"
             },
             "opera_android": {
-              "version_added": null
+              "version_added": "≤12.1"
             },
             "safari": {
-              "version_added": true
+              "version_added": "3"
             },
             "safari_ios": {
-              "version_added": true
+              "version_added": "1"
             },
             "samsunginternet_android": {
-              "version_added": true
+              "version_added": "1.0"
             },
             "webview_android": {
-              "version_added": true
+              "version_added": "1"
             }
           },
           "status": {
             "experimental": false,
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "align": {
```

Comment:
OK, was also able to verify that `<legend align=middle>foo</legend>` does center the text, so the content attribute does do something. (I suspected that maybe it was merely reflected but had no effect, but Chromium code search helped me figure it out.)